backend/dtos/lesson.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+from pydantic import Field
+
+from backend.dtos.common import BaseModel
+
+
+class LessonMinimal(BaseModel):
+    id: int = Field()
+    name: str = Field(min_length=3, max_length=1024, example="Past Simple")
+    content: str = Field(example="Past Simple is used for past!")
+
+
+class LessonDTO(BaseModel):
+    id: int = Field()
+    completed: bool = Field(False)
```

Comment:
The model of database does not contain fielt ‘completed’. Should I add this field? Why field ‘order’ does not used?

Reply:
yep, that's for purpose: in the endpoint, where we will return all the lessons, we will check if a user has a record for this lesson in the Progress table, if yes - this lesson is completed. Also we will order the lessons by Lesson.order, but we don't have to return this field to the FE, because we will return the lessons in correct order, so they won't need to sort the returned list of lessons